src/bmipy/_cmd.py

```diff
@@ -9,19 +9,29 @@
 from bmipy._version import __version__
 
 
-def main(args: tuple[str, ...] | None = None) -> int:
+def main(argv: tuple[str, ...] | None = None) -> int:
     """Render a template BMI implementation in Python for class NAME."""
     parser = argparse.ArgumentParser()
     parser.add_argument("--version", action="version", version=f"bmipy {__version__}")
     parser.add_argument("name")
```

Comment:
I'm sorry, I wasn't clear. Here's what I was thinking:
```
parser.add_argument("name", help="The BMI class name")
```

Reply:
Ah, thank you. That makes sense. I've made the change.